pkg/e2e/verify/storage.go

```diff
@@ -18,17 +26,284 @@ func init() {
 	alert.RegisterGinkgoAlert(storageTestName, "SD-SREP", "Christoph Blecker", "sd-cicd-alerts", "sd-cicd@redhat.com", 4)
 }
 
+const (
+
+	// podStartTimeout is how long to wait for the pod to be started.
+	podStartTimeout = 3 * time.Minute
+
+	// poll is how often to poll pods, nodes and claims.
+	poll = 1 * time.Second
+)
+
 var _ = ginkgo.Describe(storageTestName, func() {
 	h := helper.New()
-	ginkgo.It("should be able to be expanded", func() {
-		scList, err := h.Kube().StorageV1().StorageClasses().List(context.TODO(), metav1.ListOptions{})
-		Expect(err).NotTo(HaveOccurred(), "couldn't list StorageClasses")
-		Expect(scList).NotTo(BeNil())
-
-		for _, sc := range scList.Items {
-			Expect(sc.AllowVolumeExpansion).To(Not(BeNil()))
-			Expect(*sc.AllowVolumeExpansion).To(BeTrue())
-		}
 
-	}, 300)
+	ginkgo.Context("storage", func() {
+
+		ginkgo.It("create PVCs", func() {
+
+			sc, err := h.Kube().StorageV1().StorageClasses().List(context.TODO(), metav1.ListOptions{})
+			Expect(err).NotTo(HaveOccurred(), "couldn't list StorageClasses")
+			list, provider := getScNames(sc, h)
+			log.Printf(" PROVIDER:  %v", provider)
+			var pvc PersistentVolumeClaimConfig
+			namespace := "test-namespace"
+			_, err = createNamespace(namespace, h)
+			Expect(err).NotTo(HaveOccurred(), "couldn't create Namespace")
+			defer func() {
+				deleteNamespace(namespace, true, h)
+			}()
+			for _, name := range list {
+				var vc []*corev1.PersistentVolumeClaim
+				storageClassName := name
+				name = "pvc-osde2e-" + name
+				pvcYaml := makePersistentVolumeClaim(pvc, namespace, storageClassName, name)
+				pvc, err := createPVC(h, namespace, pvcYaml)
+				defer func() {
+					deletePersistentVolumeClaim(h, name, namespace)
+				}()
+				Expect(err).NotTo(HaveOccurred(), "couldn't create PVC")
+				vc = append(vc, pvc)
+				pod, err := createTestPod(h, namespace, vc, false)
+				Expect(err).NotTo(HaveOccurred())
+				podName := pod.GetName()
+				defer func() {
+					deletePod(podName, namespace, h)
+				}()
+
+			}
+
+		})
+
+	})
+
+	ginkgo.Context("sc-list", func() {
+		ginkgo.It("should be able to be expanded", func() {
+			scList, err := h.Kube().StorageV1().StorageClasses().List(context.TODO(), metav1.ListOptions{})
+			Expect(err).NotTo(HaveOccurred(), "couldn't list StorageClasses")
+			Expect(scList).NotTo(BeNil())
+
+			for _, sc := range scList.Items {
+				Expect(sc.AllowVolumeExpansion).To(Not(BeNil()))
+				Expect(*sc.AllowVolumeExpansion).To(BeTrue())
+			}
+
+		}, 300)
+	})
 })
+
+//Extracts Storage Class names and
+func getScNames(list *v1.StorageClassList, h *helper.H) ([]string, string) {
+	var scs []string
+	var provisioner string
+	provisioner = ""
+	provider := "gcp"
```

Comment:
i'd think we'd want to default to "aws" here?

Reply:
I  removed the default provider, now we don't default to anything.